src/main/java/org/folio/edge/inventory/service/EcsLocationsService.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.edge.inventory.service;
+
+
+import static org.folio.edge.inventory.models.InventoryViewJsonFields.TOTAL_RECORDS;
+
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.folio.edge.inventory.client.SearchClient;
+import org.folio.edge.inventory.util.JsonConverter;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class EcsLocationsService {
+  private final SearchClient searchClient;
+  private final JsonConverter jsonConverter;
+  public static final String LOCATIONS = "locations";
```

Comment:
public static variable should be declared before non-static (on top)

Reply:
fixed